WordPress/src/main/java/org/wordpress/android/ui/uploads/AbstractUploadHandler.java

```diff
@@ -0,0 +1,7 @@
+package org.wordpress.android.ui.uploads;
+
+abstract class AbstractUploadHandler {
+    abstract void unregister();
+    abstract boolean hasInProgressUploads();
+    abstract void cancelInProgressUploads();
+}
```

Comment:
Replaced the abstract class with an interface in 87c3faa, dropped `unregister()` from the interface in 8771408.

I also refactored things and adding a generic `upload()` method to the interface in 7ecfd16.